metaflow/runtime.py

```diff
@@ -241,6 +288,11 @@ def execute(self):
         # assert that end was executed and it was successful
         if ("end", ()) in self._finished:
             self._logger("Done!", system_msg=True)
+        elif self._clone_only:
+            self._logger(
+                "Clone-only resume complete -- some steps did not execute!",
```

Comment:
No steps should have executed?

Reply:
I will rephrase but basically it didn't reach end. So the steps that "executed" are only clones.